db/migrate/20160817125655_reset_override_params.rb

```diff
@@ -3,7 +3,7 @@ class LookupKey < ::ActiveRecord::Base
   end
 
   def up
-    LookupKey.where(override: false).update_all(
+    PuppetclassLookupKey.where(override: false).update_all(
```

Comment:
I'm not sure this would use the fake class correctly, might be better to use `LookupKey.where(override: false, type: 'PuppetclassLookupKey')...`